docs/farming-&-staking/farming/space-acres/space-acres-install.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: Install
+sidebar_position: 2
+description: How to run a Subspace Network Farmer with Space Acres GUI Application
+keywords:
+    - Farmer
+    - Farming
+    - GUI
+    - Executable
+---
+
+import DownloadButtons from './platforms/_download.mdx'
+
+## Get Started with Advanced CLI Installation
+
+## Step 1: Download the Space-Acres Executable
+
+<DownloadButtons />
+
+## Step 2: Install Space-Acres
+
+1. Open the downloaded Space-Acres executable to launch the Setup Wizard. This wizard will guide you through the installation process.
+    
+    ![space-acres-install-1](/img/doc-imgs/space-acres/space-acres-install-1.png) 
+
+2. Choose your installation preferences, such as the destination folder and whether to create shortcuts. You can also select to have Space Acres launch at startup.
+
+    ![space-acres-install-2](/img/doc-imgs/space-acres/space-acres-install-2.png)
+
+3. Review your settings and click `Install` to commence the installation. The wizard will then install Space Acres on your system.
+
+    ![space-acres-install-3](/img/doc-imgs/space-acres/space-acres-install-3.png)
+
+4. The installation progress will be displayed, showing the status of new files being copied.
+
+    ![space-acres-install-4](/img/doc-imgs/space-acres/space-acres-install-4.png)
+
+5. Once the installation is complete, you will see a completion screen where you can choose to run Space Acres immediately by checking the box before clicking Finish.
+
+    ![space-acres-install-5](/img/doc-imgs/space-acres/space-acres-install-5.png)
+
+## Step 3: Initalize Space-Acres
+
+1. The welcome screen provides an overview of the necessary prerequisites for running Space Acres effectively. Make sure you meet all listed requirements before proceeding. For a comprehensive list of prerequisites and explanations, refer to our [prerequisites guide.](./space-acres-prerequisites.mdx)  
```

Comment:
Prerequisite link will need changing when we merge them. Not reported in version-latest.

Reply:
Fixed in Commit [49564d5](https://github.com/subspace/subspace-docs/pull/447/commits/49564d5c02530b74351695b23e275f6ff72ea835)